.github/workflows/deploy.yml

```
@@ -4,6 +4,10 @@ on:
   push:
     branches:
       - master
+  schedule:
+      # * 6:37 every day – 37 just to not trigger exactly at 6:30 like many other;
+      # 6 is a bit before the beginning of the day in the US.
+      - cron:  '6 37 * * *'
```

This looks like it's the wrong way around: https://docs.github.com/en/actions/reference/events-that-trigger-workflows#schedule

https://crontab.guru/#6_37_*_*_*

---

```suggestion
      - cron:  '37 6  * * *' 
```

Darn, cron should take named arguments.

---

Comment is also still wrong I think? First link about says the time is in UTC, which means it'll be around midnight in US time zones.

---

Ok, move to 10:37 - 6:37EST – 3:30PST. I think that does not matter much as long as it's at night.
10:30  is still early enough for Europe I think, and we can still tweak.